addons/pharma/ACE_Medical_Treatment.hpp

```diff
@@ -27,6 +27,8 @@ class ACE_ADDON(Medical_Treatment) {
         // Reduction of damage from wounds
         opioidRelief = 1;
 
+        opioidEffect = 0;
```

Comment:
I probably missed it somewhere in the CR, but what does this do?

Reply:
:kappa: its used in feedback addon to handle chromatic abberation of ketamine/fentanyl/pervitin